hbw/production/gen_hbw_decay.py

```diff
@@ -16,7 +16,7 @@
 def gen_hbw_decay_products(self: Producer, events: ak.Array, **kwargs) -> ak.Array:
     """
     Creates column 'gen_hbw_decay', which includes the most relevant particles of a HH->bbWW(qqlnu) decay.
-    All sub-fields are individual GenParticles, except 'foo', which is an array of all non-Higgs 
+    All sub-fields are individual GenParticles, except 'foo', which is an array of all non-Higgs
```

Comment:
We should switch to something else instead of 'foo', even if it might not be the optimal/final choice.
How about `byproducts`? or simply `other`?